public/index.html

```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
     </div>
   </div>
 </div>
+<!-- HCA Application End -->
 
+<!-- Maintenance Page Start -->
+  <!--<div class="splash">
+    <div class="row">
+      <div class="small-12 columns">
+        <div class="pitch">
+          <h2 class="tagline"><span>Wednesday, June 29th, 2016 at 3:38 PM EST</span></h2>
```

Comment:
@maryannbrody -- Do we think it is crucial for the user to know the time? I am thinking about removing it from the header. I started looking through other examples of system maintenance messages and very few have a timestamp.